mcdemoaux/vision/vis.py

```diff
@@ -22,10 +22,6 @@ def run(self):
             while self.queue:
                 with self._lock:
                     data = self.queue.pop()
-                    if len(data[1].shape) == 3:
-                        image = cv2.cvtColor(data[1], cv2.COLOR_BGR2RGB)
```

Comment:
Why do you think it's unnecessary?

Reply:
Or do we have now new format and new conversion with new image transfer protocol?

Reply:
@CICS-Oleg  well, I'm not sure actually. Does we need that if-statement? Currently bgr2rgb done in timestamped_video_frame in my PR https://github.com/trueagi-io/minecraft-demo/pull/136/files which still waiting for your approval =)